draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -506,8 +506,12 @@ of flows.
 
 ## QUIC Streams {#quic-streams}
 
-To send RTP/RTCP packets over QUIC streams, a sender MUST open a
-new unidirectional QUIC stream. Streams are unidirectional because there is no
+To send RTP/RTCP packets over QUIC streams, a sender MUST open at least one new unidirectional QUIC stream.
+In order to permit QUIC streams to open, a RoQ sender SHOULD configure non-zero minimum values for the number of permitted streams and the initial stream flow-control window, based on the number of parallel, or simultaneously active, RTP/RTCP flows.
```

Comment:
This is a useful addition, but I suggest moving it into the MAX_STREAMS/flow control subsection to keep all the things related to that issue in one place.

Reply:
Agreed (thanks for the helpful suggestion). I committed a change for this.